### ISSUE TITLE
Feature request: Specify a directory that isn't root

### ISSUE THREAD

**Issue by lucywyman** [igrigorik/bugspots#36]
I have a large project, where I only want to see bug spots in one of the subdirectories, but when I tried running `bugspots -b mybranch ./directory` I got the error `Invalid Git repository - please run from or specify the full path to the root of the project.`.  I can look into adding this feature next week, but was wondering if anyone else thinks this would be useful? Is this repo open to outside contributors?

**Comment by igrigorik** [comment]
Yep, makes sense! In terms of plumbing, I guess you'd need to filter the git history for files that are only in that directory..
